grafana/scylla-detailed.master.template.json

```diff
@@ -465,10 +465,19 @@
                         "title": "Writes currently blocked on commitlog"
                     },
                     {
-                        "class": "text_panel",
-                        "content": "",
-                        "mode": "markdown",
-                        "span": 3
+                        "class": "graph_panel",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "clamp_max(1 + sum(rate(scylla_cache_row_hits[1m])) by ([[by]])/(sum(rate(scylla_cache_row_misses[1m])) by ([[by]]) + 0.00001),100)",
+                                "intervalFactor": 1,
+                                "legendFormat": "",
+                                "refId": "A",
+                                "step": 1
+                            }
+                        ],
+                        "description" :"Reciprocal Miss Rate is a score in the range of 1 to 100 that is used to optimized query distribution.\n\nThe higher the score, the more likely the data will be in the cache and the more queries this node will get.\n\nThis is an aggregate estimation of the true score.",
```

Comment:
As a user, I'm pretty sure I wouldn't have understood what this says.
First, why not say explicitly what this value calculates? You can say that"cache miss rate" is the fraction of queries that can't be answered from the row cache and need to go to disk, and "reciprocal miss rate" is the reciprocal (1/x) of that.
Second, the "used to optimized query distribution" (by the way, you have a grammar error there, should be "optimize", not optimized) is a bit vague. You say "the higher the score..." but it's not clear how, or when. How about something like "This metric is used to decide the fraction of read requests to send to each replica - a replica with twice the RMR value of another replica will serve twice the number of read requests."
Third, I don't know what the last sentence means (what is an "aggregate estimation" and what is the "true score"?). Maybe (?) you meant that you use (if I understand correctly) an "average" rate (in the sense of Cauchy's theorem, https://en.wikipedia.org/wiki/Mean_value_theorem) from the last one minute (the 1m in your formula) and isn't the current instantaneous rate?
Finally, I doubt that any description of this length will be completely self-explanatory. I think it's time we start considering much longer descriptions, which either appear here, or on our web site with a link from here.

Reply:
The "real" score is calculated per table, it's actually reported for users that choose to open per-table metrics.
This is an estimation that tries to guess on the shard (or node) level, how much more traffic a node would get.

And thinking from a trouble-shooting perspective, you see a node that gets half the traffic and you try to understand why, this can give you a clue.

Thanks for the suggestion about the description, I will make use of that.

About general documentation, you are totaly right, and infact starting from the comming version (3.7) I'm planning to start add specific documentation, the first phase will be for Advisor and Alerts, but definitely Adding per panels documentation would follow.